frontend/packages/knative-plugin/src/topology/knative-topology-utils.ts

```diff
@@ -542,7 +796,10 @@ export const getTriggerTopologyEdgeItems = (broker: K8sResourceKind, resources):
           data: {
             resources: {
               obj: trigger,
-              connections: [broker, knativeService],
+              eventSources: getSubscribedEventsources(broker, resources),
+              brokers: [broker],
+              ksservices: [knativeService],
```

Comment:
Why are we making these array?

Reply:
we have a common component for listing the knative resources, `PubSubResourceOverviewList` This components works based on the array, so  I am maintaing the same here in above usecases for edges as well instead of altering the component and type for accepting object/array.